product/models.py

```diff
@@ -0,0 +1,125 @@
+from django.db import models
+
+class Product(models.Model):
+    name           = models.CharField(max_length = 100)
+    price          = models.DecimalField(max_digits = 10, decimal_places = 2)
+    description    = models.CharField(max_length = 100)
+    main_image_url = models.URLField()
+    category       = models.ManyToManyField('category', through = 'productcategory')
+    item           = models.ManyToManyField('item', through = 'productitem')
+    type           = models.ManyToManyField('type', through = 'producttype')
+    Technology     = models.ManyToManyField('technology', through = 'producttechnology')
+    color          = models.ManyToManyField('color', through = 'productcolor')
+    size           = models.ManyToManyField('size', through = 'productsize')
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        db_table = "products"
+
+class SubImageUrl(models.Model):
```

Comment:
class명이 Image로 충분합니다.
```suggestion
class Image(models.Model):
```